pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -41,18 +42,23 @@ func (f *frontend) putOrPatchOpenShiftCluster(w http.ResponseWriter, r *http.Req
 	subId := chi.URLParam(r, "subscriptionId")
 	resourceProviderNamespace := chi.URLParam(r, "resourceProviderNamespace")
 
+	identityUrlHeader := "x-ms-identity-url"
```

Comment:
nit: did a quick search and we don't generally create constants for headers we set/get, might be worth considering as a later refactoring effort.